### PR TITLE
UhOh365: Initial commit

### PR DESCRIPTION
**Pull request by sachinkamath**
This PR adds PKGBUILD for https://github.com/Raikia/UhOh365

### CODE REVIEW

**Comment by Edu4rdSHL** [comment]
Hello, thanks. Have you tested it PKGBUILD? Because looking so far the pkgname is lowercase but when you are cloning the repo it creates a folder with lower and uppercase letters, so the cd $pkgname will not work.

File: packages/uhoh365/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+pkgver=24.bc22611
+pkgrel=1
+pkgdesc='Script to enumerate Office 365 users without performing login attempts'
+groups=('blackarch' 'blackarch-recon')
+arch=('any')
+url='https://github.com/Raikia/UhOh365'
+license=('GPL-3.0')
+depends=('python' 'python-requests')
+makedepends=('git')
+source=('git+https://github.com/Raikia/UhOh365.git')
```
Comment (Edu4rdSHL) [inline packages/uhoh365/PKGBUILD]: Use `"git+https://github.com/Raikia/$pkgname.git"`.

**Comment by sachinkamath** [comment]
> Hello, thanks. Have you tested it PKGBUILD? Because looking so far the pkgname is lowercase but when you are cloning the repo it creates a folder with lower and uppercase letters, so the cd $pkgname will not work.

I had it handled in prepare() but looks like `_pkgname` is the way to go :)

Will keep it innmind.

**Comment by Edu4rdSHL** [comment]
Hey man, yes, in that case you should use `_pkgname`, very practical without need to do unneeded steps. Thank you for the PR man, we really appreciate it and hope you continue doing it.

**Pull request merged**